src/components/UI/Card.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { VscEdit } from "react-icons/vsc";
+import { VscSave } from "react-icons/vsc";
+import { VscClose } from "react-icons/vsc";
+//import classnames from 'classnames';
+
+import "./Card.css";
+
+var classNames = require("classnames");
+
+class Card extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: false,
+      clicked: false,
+        input: "",
+        inputValue: "",
+        textarea: "",
+        textareaValue: "",
+    };
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.setState({ input: this.state.inputValue });
+    this.setState({ textarea: this.state.textareaValue });
+  }
```

Comment:
Move this out to App component along with actual card data. Pass via props.
No need to call setState for each property: you can use smth like 
data: {...tempData}.
In that way you won't need to change that part if any properties added to objects.